### PR TITLE
Fix Get File from S3

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Additional context
fixes https://github.com/solop-develop/frontend-core/issues/1223

### CODE REVIEW

File: src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java [hunk 1]
```diff
@@ -439,27 +444,39 @@ public static ListEntitiesResponse.Builder listFilePreview(ListFilePreviewReques
...
 			charsetValue = Charset.defaultCharset().name();
 		}
 		Charset charset = Charset.forName(charsetValue);
+		MClientInfo clientInfo = MClientInfo.get(Env.getCtx());
+		// Connector S3
+		MADAppRegistration genericConnector = MADAppRegistration.getById(
+			Env.getCtx(),
+			clientInfo.getFileHandler_ID(),
+			null
+		);
```
Comment (EdwinBetanc0urt) [inline src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java:466]: please validate `genericConector` and launch exception with 
```java
if (genericConnector == null || genericConnector.getAD_AppRegistration_ID() <= 0) {
    throw new AdempiereException("@AD_AppSupport_ID@ @NotFound@");
}
```

File: src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java [hunk 2]
```diff
@@ -439,27 +444,39 @@ public static ListEntitiesResponse.Builder listFilePreview(ListFilePreviewReques
...
+		//	Load
+		IAppSupport supportedApi = AppSupportHandler.getInstance().getAppSupport(genericConnector);
```
Comment (EdwinBetanc0urt) [inline src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java:468]: please validate `supportedApi`
```java
if (supportedApi == null) {
	throw new AdempiereException("@AD_AppSupport_ID@ @NotFound@");
}
if (!IS3.class.isAssignableFrom(supportedApi.getClass())) {
	throw new AdempiereException("@AD_AppSupport_ID@ @Unsupported@");
}
```

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java (hunk 1 above)
Comment (elsiosanchez) [reply src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java:469]: add validations https://github.com/solop-develop/adempiere-grpc-server/pull/856/commits/3f3b29d589036557e935a0e1a4a25a05dd8d15b4

File: src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java (hunk 2 above)
Comment (elsiosanchez) [reply src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java:474]: add validations https://github.com/solop-develop/adempiere-grpc-server/pull/856/commits/3f3b29d589036557e935a0e1a4a25a05dd8d15b4

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**